recipes/pyemojify/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
you might consider putting this under the build section as:
```
build:
   script: python setup.py install --single-version-externally-managed --record record.txt
```

Reply:
Does this mean I wouldn't need a build.sh file then?

Reply:
Yes, you can look at the example file for an example of this.